tests/tests_query_builders/test_qb2.py

```diff
@@ -375,6 +375,23 @@ def test_where_rel_2(qbr):
     assert data_binding_dict == {}
 
 
+def test_where_rel_3(qbr):
+    test_map = {
+        'nobs': {
+            'EXISTS>': {'include': ["'Test' : {'uri': ['neo4j://graph.schema#Term/S980028/S91301']}"]},
```

Comment:
@PranjaliParnerkar 
this is supposed to be 
`'EXISTS>': {'include': [{'Test' : {'uri': ['neo4j://graph.schema#Term/S980028/S91301']}}]}`
not in the quotes.

Suggest to do literal_eval outside of the function when reading the property from the SUBLASS_OF relationship,
otherwise it is not transparent which structure does a used of the function is supposed to pass here